crates/runtime/src/helpers.c

```diff
@@ -27,10 +27,12 @@ typedef jmp_buf platform_jmp_buf;
 // runtime) is assuming is callee-saved.
 //
 // [GCC docs]: https://gcc.gnu.org/onlinedocs/gcc/Nonlocal-Gotos.html
-// [clang docs]: https://llvm.org/docs/ExceptionHandling.html#llvm-eh-sjlj-setjmp
+// [clang docs]:
+// https://llvm.org/docs/ExceptionHandling.html#llvm-eh-sjlj-setjmp
```

Comment:
Looks like this may be becoming a broken link

Reply:
Had to disable the formating to get this right.